### PR TITLE
Add suffixes to typos that start with "f"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18202,7 +18231,13 @@ fewg->few, fugue,
 fewsha->fuchsia
 fezent->pheasant
 fhurter->further
+fial->fail
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:18234]: Perhaps:
```
fial->fail, dial, vial, final,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18216,7 +18251,9 @@ fied->field
 fiedl->field
 fiedled->fielded
 fiedls->fields
+fieds->fields
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:18254]: Although uncommon, perhaps:
```
fieds->fields, fiends,
```
Would apply to singular too.

**Review by DimitriPapadopoulos — COMMENTED** [review]
A couple suggestions.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt]: Done. I also added `fially->finally` and `fialog->dialog`+friends

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**